deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -2203,28 +2212,52 @@ def regular_file_cud(folder, log_monitor, file_list=['testfile0'], time_travel=F
         create_file(REGULAR, folder, name, content=content)
 
     check_time_travel(time_travel, monitor=log_monitor)
-    event_checker.fetch_and_check('added', min_timeout=min_timeout, triggers_event=triggers_event)
+
+    event_checker.fetch_and_check('added', event_mode=event_mode, min_timeout=min_timeout,
+                                  triggers_event=triggers_event)
     if triggers_event:
         logger.info("'added' {} detected as expected.\n".format("events" if len(file_list) > 1 else "event"))
 
+        if time_travel:
+            log_monitor.start(timeout=global_parameters.default_timeout, callback=callback_detect_end_scan,
+                              update_position=True,
+                              error_message=f"End of scheduled scan not detected after "
+                              f"{global_parameters.default_timeout} seconds")
+
     # Modify previous text files
     for name, content in file_list.items():
         modify_file(folder, name, is_binary=isinstance(content, bytes))
 
     check_time_travel(time_travel, monitor=log_monitor)
-    event_checker.fetch_and_check('modified', min_timeout=min_timeout, triggers_event=triggers_event, extra_timeout=2)
+
+    event_checker.fetch_and_check('modified', event_mode=event_mode, min_timeout=min_timeout,
+                                  triggers_event=triggers_event)
```

Comment:
Same as before

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1474/commits/1b7ea0a207fdaf1ba67ee56d13785ac575b1ae0a)